app/controllers/invites_controller.rb

```diff
@@ -34,4 +44,10 @@ def redeem
     invite.course.students << current_user
     render status: :ok, json: invite
   end
+
+  def unused
```

Comment:
For this kind of filtering query, we could possibly make use of scopes in our ActiveRecord models. This can come in another PR.